index.js

```diff
@@ -63,7 +71,13 @@ try{
 }
 
 try{
-	console.log(vm.run(untrusted4));
+	console.log(vm.run(untrusted4).addr);
```

Comment:
Why did you change code of the Balancer ???

Reply:
Because that's what the real FB does.